security-evidence.md

```diff
@@ -61,134 +61,134 @@
 - /v1/query POST succeeds after reconnect to DB: [FailureTests.scala](ledger-service/http-json/src/failurelib/scala/http/FailureTests.scala#L260)
```

Comment:
All changes in this file are auto generated from changes to EvaluationOrderTest?